qa-pipelines/flags.yml

```diff
@@ -4,37 +4,39 @@
 # The settings in this file will also be valid for our most complete pipeline
 # (which performs an upgrade and all tests)
 
-# deploy cf for upgrade pipelines
-enable-cf-deploy-pre-upgrade: true
+# Report status as steps complete
+status-reporting: true
+
+# version to upgrade from; if null, upgrade is not performed.
+upgrade-from-version: scf-sle-2.15.2+cf3.6.0.0.gde1bd02f
```

Comment:
I suspect renaming `enable-cf-deploy-pre-upgrade` to this will lead to accidental enabling on pipelines where that task is not intended to be enabled